### ISSUE TITLE
Greedy Cuts fails to cut a 1x1 matrix

### ISSUE THREAD

**Issue by ariasanovsky** [ariasanovsky/signtensors#5]
C.f. `cargo test sct_of_all_ones_1_by_1_matrix`

**Comment by ariasanovsky** [comment]
Yields

```shell
running 1 test
test sct::tests::sct_of_all_ones_1_by_1_matrix ... FAILED

failures:

---- sct::tests::sct_of_all_ones_1_by_1_matrix stdout ----
[src/sct.rs:480:9] &one_cut.sct.c = [
    0.0,
]
[
[0.0],
]
thread 'sct::tests::sct_of_all_ones_1_by_1_matrix' panicked at src/sct.rs:482:9:
Assertion failed at src/sct.rs:482:9
Assertion failed: one_cut.remainder_cis.norm_l2() < 1e-5
- one_cut.remainder_cis.norm_l2() = 1.0
- 1e-5 = 1e-5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    sct::tests::sct_of_all_ones_1_by_1_matrix

test result: FAILED. 0 passed; 1 failed; 0 ignored; 0 measured; 7 filtered out; finished in 0.00s

error: test failed, to rerun pass `--lib`
```

@sarah-quinones

**Comment by sarah-quinones** [comment]
fixed by 9340288
